### DISCUSSION TITLE
Failed to initialize lotus-miner: creating miner failed

### DISCUSSION THREAD

**Discussion by whhpc19891120** [filecoin-project/lotus#7766] [Build and Sync]
while i execute this command i have a problem bow,  what can i do then, thank you 

lotus-miner init --owner=f3ryvywonhcgpmyw5ondjl4puxzieh6xo7deub6bqdc6xlkyp3g3ol4cum5jecb7s52bdefwbpcwj5zcflcasq  --worker=f3xgmgwmajftgbt6kqoh3wyfbgms3izh6afyzbbhgqgs34fv7ig6ghcvy2ig5prb5jvookwenqyoqmhxqucn7q --no-local-storage

logs
################################################################################################

{"level":"info","ts":"2021-08-30T10:13:59.579+0800","logger":"main","caller":"lotus-miner/init.go:154","msg":"Trying to connect to full node RPC"}
{"level":"info","ts":"2021-08-30T10:13:59.800+0800","logger":"main","caller":"lotus-miner/init.go:166","msg":"Checking full node sync status"}
{"level":"info","ts":"2021-08-30T10:17:50.805+0800","logger":"main","caller":"lotus-miner/init.go:174","msg":"Checking if repo exists"}
{"level":"info","ts":"2021-08-30T10:17:50.805+0800","logger":"main","caller":"lotus-miner/init.go:190","msg":"Checking full node version"}
{"level":"info","ts":"2021-08-30T10:17:50.806+0800","logger":"main","caller":"lotus-miner/init.go:201","msg":"Initializing repo"}
{"level":"info","ts":"2021-08-30T10:17:50.806+0800","logger":"repo","caller":"repo/fsrepo.go:140","msg":"Initializing repo at '/root/.lotusminer'"}
{"level":"info","ts":"2021-08-30T10:17:50.807+0800","logger":"main","caller":"lotus-miner/init.go:416","msg":"Initializing libp2p identity"}
{"level":"info","ts":"2021-08-30T10:17:50.877+0800","logger":"badger","caller":"v2@v2.2007.2/levels.go:183","msg":"All 0 tables opened in 0s\n"}
{"level":"info","ts":"2021-08-30T10:17:50.890+0800","logger":"badger","caller":"v2@v2.2007.2/levels.go:183","msg":"All 0 tables opened in 0s\n"}
{"level":"error","ts":"2021-08-30T10:19:36.544+0800","logger":"main","caller":"lotus-miner/init.go:261","msg":"Failed to initialize lotus-miner: creating miner failed:\n    main.storageMinerInit\n        /kingdom/lotus-v1.11.2/lotus/cmd/lotus-miner/init.go:541\n  - push worker init:\n    main.createStorageMiner\n        /kingdom/lotus-v1.11.2/lotus/cmd/lotus-miner/init.go:655\n  - taking lock: context canceled"}
{"level":"info","ts":"2021-08-30T10:19:36.544+0800","logger":"main","caller":"lotus-miner/init.go:266","msg":"Cleaning up /root/.lotusminer after attempt..."}

**Comment by Reiers** [comment]
I was able to init the miner just fine on the latest release. 
`lotus version 1.13.2-rc4+mainnet+git.e05fdf892
`
Moving this issue to discussion for troubleshooting.

**Comment by LexLuthr** [comment]
@whhpc19891120 Do you still need help with this error?

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
